bindata/scripts/kargs.sh

```diff
@@ -5,54 +5,83 @@ command=$1
 shift
 declare -a kargs=( "$@" )
 ret=0
-args=$(chroot /host/ cat /proc/cmdline)
 
-IS_OS_UBUNTU=true; [[ "$(chroot /host/ grep -i ubuntu /etc/os-release -c)" == "0" ]] && IS_OS_UBUNTU=false
+if [[ -d "/bindata/scripts" ]];then
+    chroot_path="/host/"
+else
+    chroot_path="/"
+fi
+
+args=$(chroot "$chroot_path" cat /proc/cmdline)
+
+IS_OS_UBUNTU=true; [[ "$(chroot "$chroot_path" grep -i ubuntu /etc/os-release -c)" == "0" ]] && IS_OS_UBUNTU=false
```

Comment:
you can:

```
. /etc/os-release
if [[ $ID == "ubuntu" ]]; then
..
fi
```

Reply:
done

Reply:
rolled back to a previous solution. this doesn't work for chroot